lib/index.js

```diff
@@ -91,7 +94,11 @@ export default function retextSmartypants(options) {
     methods.push(quotesDefault)
   }
 
-  if (settings.ellipses !== false) {
+  if (settings.ellipses === 'unspaced') {
+    methods.push(ellipsesUnspaced)
+  } else if (settings.ellipses === 'spaced') {
+    methods.push(ellipsesSpaced)
+  } else if (settings.ellipses !== false) {
```

Comment:
```suggestion
  if (settings.ellipses === 'spaced') {
    methods.push(ellipsesSpaced)
  } else if (settings.ellipses === 'unspaced') {
    methods.push(ellipsesUnspaced)
  } else if (settings.ellipses) {
```

Sorting. Simplify superfluous condition

Reply:
@wooorm The `!== false` check is actually necessary here. I re-added it and the tests pass now.